caffe2/onnx/backend.cc

```diff
@@ -548,50 +548,12 @@ Caffe2Ops Caffe2Backend::CreateConvPoolOpBase(
 Caffe2Ops Caffe2Backend::CreatePadPool(
     OnnxNode* onnx_node,
     const ConversionContext& ctx) {
-  auto& node = onnx_node->node;
-  auto& attributes = onnx_node->attributes;
   Caffe2Ops ret;
   // Pad
   bool padding = false;
```

Comment:
Let's also remove this logic.

Reply:
Removing the entire CreatePadPool function and use CreateConvPoolOpBase directly for AveragePool and GlobalAveragePool